docs/api/flink/Constructor.md

```diff
@@ -49,22 +103,43 @@ Format:
 Since: `v1.3.0`
 
 SQL example:
+
 ```sql
-SELECT ST_GeomFromKML('<LineString><coordinates>-71.1663,42.2614 -71.1667,42.2616</coordinates></LineString>') AS geometry
+SELECT ST_GeomFromKML('
+	<LineString>
+		<coordinates>
+			-71.1663,42.2614 
+			-71.1667,42.2616
+		</coordinates>
+	</LineString>
+')
+```
+
+Output:
+
+```
+LINESTRING (-71.1663 42.2614, -71.1667 42.2616)
 ```
 
 ## ST_GeomFromText
 
-Introduction: Construct a Geometry from Wkt. Alias of  [ST_GeomFromWKT](#ST_GeomFromWKT)
+Introduction: Construct a Geometry from WKT. Alias of  [ST_GeomFromWKT](#ST_GeomFromWKT)
 
 Format:
 `ST_GeomFromText (Wkt:string)`
 
 Since: `v1.2.1`
 
 SQL example:
+
 ```sql
-SELECT ST_GeomFromText('POINT(40.7128 -74.0060)') AS geometry
+SELECT ST_GeomFromText('POINT(40.7128 -74.0060)')
+```
+
+Output:
+
+```
+POINT(40.7128 -74.006)
 ```
 
 ## ST_GeomFromWKB
```

Comment:
Please specify that this function supports EWKB format. Example input: `01010000a0e6100000000000000000f03f000000000000f03f000000000000f03f`

Please use ST_asEWKT to print it.